### DISCUSSION TITLE
FEATURE REQUEST

### DISCUSSION THREAD

**Discussion by GeorgeMuigaiNjau** [microsoft/vscode-discussions#2205] [Extension Development QnA] [answered]
I don't know if I'm asking on the right place, but I think if this works it's worth it; I was thinking, what if there is something that could be done such that you can open a project from GitHub remotely, edit live changes saved for you and automated commits, the point is just been able to work on a project on git, remotely and changes pushed automatically

**Comment by alefragnani** [comment, marked as answer]
Hi @George-9 ,

You already can do that, using the [GitHub Repositories](https://marketplace.visualstudio.com/items?itemName=GitHub.remotehub) extension. More details here https://code.visualstudio.com/docs/sourcecontrol/github#_github-repositories-extension

Hope this helps
